### DISCUSSION TITLE
Update RTLSDR-Airband 3.1 to 3.2.1?

### DISCUSSION THREAD

**Discussion by kf7ay** [rtl-airband/RTLSDR-Airband#249] [Q&A]
What is the best way to Update RTLSDR-Airband 3.1 to 3.2.1 on a PI3 ?

Thx,

John

**Comment by szpajder** [comment]
There are no backward-incompatible changes in 3.2.1 with respect to 3.1, so
just install the new version as you would install it for the first time.

"make install" replaces the old /usr/local/bin/rtl_airband with the new
version (if you are extra cautious, you may copy the old binary somewhere
in case you would want to go back for any reason). If you already have your
customized config file in /usr/local/etc/rtl_airband.conf, then "make
install" will preserve it. Then just restart the program, check the logs
(/var/log/messages or the like) to verify whether you are indeed running
the new version and you are done.

**Reply by kf7ay** [reply]
Sweet! Thank you szajader. Is NFM enabled by default with 3.2.1? 

John

**Comment by szpajder** [comment]
It's not. You have to enable it explicitly.
